dim-web/src/routes/websocket.rs

```diff
@@ -0,0 +1,259 @@
+use std::collections::HashMap;
+use std::hash::Hash;
+use std::net::SocketAddr;
+use std::pin::Pin;
+
+use tokio::sync::mpsc::{unbounded_channel, UnboundedReceiver, UnboundedSender};
```

Comment:
For style I prefer the imports to be broken down.
```suggestion
use tokio::sync::mpsc::unbounded_channel;
use tokio::sync::mpsc::UnboundedReceiver;
use tokio::sync::mpsc::UnboundedSender;
```